tests/tck/features/bugfix/MatchUsedInPipe.feature

```diff
@@ -75,14 +77,11 @@ Feature: Test match used in pipe
       """
       MATCH (n:player{name:"Tim Duncan"})-[]-(m) RETURN n,m | GO FROM $-.n OVER *;
       """
-    Then a SemanticError should be raised at runtime: `$-.n', the srcs should be type of FIXED_STRING, but was`__EMPTY__'
+    Then a SyntaxError should be raised at runtime: syntax error near `| GO FRO'
 
   Scenario: Set op after match
     When executing query:
       """
       MATCH (n:player{name:"Tim Duncan"}) RETURN n UNION MATCH (n:player{name:"Tony Parker"}) RETURN n
       """
-    Then the result should be, in any order, with relax comparison:
-      | n               |
-      | ("Tim Duncan")  |
-      | ("Tony Parker") |
+    Then a SyntaxError should be raised at runtime: syntax error near `UNION'
```

Comment:
This is not right, MATCH ... UNION MATCH ... is a legal cypher syntax.

Reply:
done